pom.xml

```diff
@@ -43,13 +47,11 @@
 
     <properties>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <confluent.common.version>3.2.1-SNAPSHOT</confluent.common.version>
-        <slf4j.version>1.7.21</slf4j.version>
```

Comment:
Is this just the starting point? Or will other repos use a different parent pom (e.g. this one)? Things like jackson and the `kafka.scala.version` seem like they could be shared to avoid potential conflicts (e.g. differing jackson versions across connectors can be an issue).